### ISSUE TITLE
Linking barcodes to enhancer elements

### ISSUE THREAD

**Issue by voichek** [tobjores/tobacco-STARR-seq#1]
Dear Jores,

I am trying to understand the technicalities of how the analysis in your interesting paper was done.
Can you please direct me to how barcodes were linked to the corresponding enhancer elements:
1. Is the relevant code part of this repository? 
2. Which are the relevant SRA sequencing files that contain this information?

Best wishes,
Yoav

**Comment by tobjores** [comment]
Dear Yoav,
sorry for the late reply.
I used different strategies to link barcodes and enhancers. The full-length enhancer constructs were constructed individually and then the barcodes were sequenced using the procedure outlined in the paper.
Enhancer fragments and the variants from the saturation mutagenesis, were linked to the barcodes by paired-end sequencing the fragment/variant and using one of the index reads to sequence the barcode. The code for this step is not included in this repository as it highly depends on the library and the sequencing run layout used. There is not one solution that fits all.
Please let me know if you want help with linking barcodes and enhancers for your library.
Best,
Tobias

**Comment by voichek** [comment]
Dear Tobias,

Thank you for your answer and willingness to help.

Can you refer me please to the SRA accession (or link if deposited elsewhere) that have the paired-end data used to link the barcodes to enhancers for Figure 5 & 6. If the data was not deposited on a public repository will you be willing to share it privately?

Best wishes and thanks for your help,
Yoav

**Comment by tobjores** [comment]
Dear @voichek,
I have added the code that I used for the subassembly of the libraries in Fig. 5 & 6 to this repository (in the folder 'subassembly_files'). All other constructs were sequenced individually and the barcodes were  sequenced using the procedure outlined in the paper.
The sequencing files from the subassembly were not included in the SRA for this paper and I can't add anything to this now. They are also way to big for this repository (several GB). If you want to take a look at them, I can share them privately (let me know your email address).
Best,
Tobias
